src/var.c

```diff
@@ -250,11 +271,67 @@ static int fi_parse_bool(const char *str_value)
 	return -1;
 }
 
+static void load_conf() {
+	char *line;
+	char *tmp;
+	struct fi_conf_entry *conf;
+
+	dlist_init(&conf_list);
+
+	FILE *fptr = fopen(DEFAULT_CONF_FILE_PATH, "r");
+	if (!fptr)
+		return;
+
+	line = calloc(1, MAX_CONFIG_LINE_LENGTH);
+	if (!line)
+		goto out;
+
+	while (fgets(line, MAX_CONFIG_LINE_LENGTH, fptr) != NULL) {
+		tmp = strtok(strtok(line, "\n"), "=");
```

Comment:
Best to use strtok_r() in the off chance some other thread somewhere is invoking strtok.  Though, you could probably just use strchr() instead of strtok(), since we're searching for '=' and not overly concerned about mis-formatted data.

Reply:
good call on strchr. seems a bit cleaner and safer